security/storage/google/public_buckets/public_buckets.pt

```diff
@@ -0,0 +1,91 @@
+name "Google Open Buckets Policy"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for buckets that are open to the public"
+long_description "Version 1.0"
+severity "high"
+category "Security"
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "param_escalate_to" do
+  type "list"
+  label "Email address to send escalation emails to"
+end
```

Comment:
Any thoughts about adding additional checks for email ? 
Something like 

parameter "escalate_to" do
  type "string"
  label "Email address to send escalation emails to"
  min_length 1
  allowed_pattern /^.+\@.+\..+$/
  constraint_description <<-EOS
  Please enter a valid email user@example.com
  EOS
end

Without notifications and alerts  that the policy has failed, thinking of getting ahead of a typo or missed parameter

Reply:
We debated in PHX, not sure where we left it. I am not doing constraints on email until we decide.